index.html

```diff
@@ -0,0 +1,91 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <header id="header">
+      <div class="logo">
+        <img src="logo.png" alt="" id="header-img">
+      </div>
+    
+    <nav id="nav-bar">
+      <ul>
+        <li><a id="nav-link" href="#"> Features</a>
+        <li><a  id="nav-link" href="#"> How it works</a>
+        <li><a  id="nav-link" href="#"> Pricing</a>
+      </ul>
+      
+    </nav>
+
+    </header>
+    
+    
+    <section id="hero">
+      <h2> Handcrafted, home-made masterpieces </h2>
+      <form action="https://www.freecodecamp.com/email-submit" id="form">
+        <input id="email" placeholder="Enter your email address">
+        <input id="submit" type="submit" value="Get Started" >
+      </form> 
+    </section>
+
+    <section id="feature">
+      <div class="grid">
+        <div class="icon"><i class="fa fa-3x fa-fire"></i></div>
+        <div class="desc">
+          <h2>Premium Materials</h2>
+          <p>
+            Our trombones use the shiniest brass which is sourced locally. This
+            will increase the longevity of your purchase.
+          </p>
+        </div>
+      </div>
+    </div>
+      <div class="grid">
+        <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+        <div class="desc">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have finished
+            making it. We also provide free returns if you are not satisfied.
+          </p>
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+        </div>
+        <div class="desc">
+          <h2>Quality Assurance</h2>
+          <p>
+            For every purchase you make, we will ensure there are no damages or
+            faults and we will check and test the pitch of your instrument.
+          </p>
+        </div>
+      </div>
+    </section>
+    <section id="video">
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
+    
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
the codebase should be free of unneeded commented messages from the project source.

Reply:
- [ ] [Required] Good comment:ok_hand:. However, this also misses the 'why'. What do you think?

Reply:
the codebase should be free of unneeded commented messages from the project source. This will help to keep your code clean and more readable